src/apps/core/submodules/alerts/alerts.js

```diff
@@ -29,8 +66,32 @@ define(function(require) {
 						}
 					});
 				}
-			], function(err, data) {
-				// TODO: Alerts pulled
+			], function(err, alerts) {
+				if (err) {
+					return;
+				}
+
+				// If there is no error, re-render topbar element with alerts
+				renderTemplate(alerts);
+			});
+		},
+
+		/**
+		 * Bind template content events
+		 * @param  {Object} args
+		 * @param  {jQuery} args.template  Template to bind
+		 */
+		alertsBindEvents: function(args) {
+			var self = this,
+				$template = args.template;
+
+			$template.find('#main_topbar_alert_toggle_link').on('click', function() {
+				$(this).find('.badge').fadeOut({
```

Comment:
We generally use the [more concise](https://devdocs.io/jquery/fadeout#fadeOut-duration-complete) form of `jQuery#fadeOut`